### PR TITLE
move cli, explorer, tracker and ui and tools common files 

### PR DESCRIPTION
**Pull request by kramasamy**
into heron/tools

### CODE REVIEW

**Comment by objmagic** [comment]
this breaks since the hardcoded level is changed:
https://github.com/twitter/heron/blob/master/heron/common/src/python/utils/config.py#L130

File: heron/tools/common/src/python/utils/config.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def get_heron_dir_explorer():
   From heron-cli with modification since we need to reuse cli's conf
   :return: root location for heron-cli.
   """
```
Comment (taishi8117) [inline heron/tools/common/src/python/utils/config.py]: btw, why do we need this? Isn't this the same with `get_heron_dir()`?

**Comment by taishi8117** [comment]
heron-ui is still not working

**Comment by kramasamy** [comment]
ui, tracker, cli and explorer are working. Let me know if this PR is good to go.

**Comment by objmagic** [comment]
👍 LGTM

**Pull request merged**